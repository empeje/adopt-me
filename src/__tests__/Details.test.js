import React from "react";
import { create } from "react-test-renderer";
import Details from "../Details";

test("snpashot", () => {
  const c = create(<Details />);
  expect(c.toJSON()).toMatchSnapshot();
});
