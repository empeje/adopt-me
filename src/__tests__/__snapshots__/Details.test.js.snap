// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snpashot 1`] = `
<h1>
  Loading...
</h1>
`;
